### DISCUSSION TITLE
Is it possible to get "changing-access-permissions-for-wikis" checkbox via rest api

### DISCUSSION THREAD

**Discussion by roma8389** [community/community#36396] [Apps, API and Webhooks]
i am looking for a way to get this info
https://docs.github.com/en/communities/documenting-your-project-with-wikis/changing-access-permissions-for-wikis
via a rest api but didnt manage to find one.

**Comment by ossanna16** [comment]
Hey there @roma8389 and thank you for asking a good question! I'm glad you found our community 🙂

**Reply by roma8389** [reply]
do u have solution?

**Reply by ossanna16** [reply]
I actually don't myself as I'm not a technical expert when it comes to REST APIs but perhaps someone else here can help you!
